plugin.video.meta/resources/lib/meta/navigation/tvshows.py

```diff
@@ -100,10 +100,10 @@ def tv_search():
     """ Activate movie search """
     search(tv_search_term)
 
-@plugin.route('/tv/play_by_name/<name>/<season>/<episode>/<lang>')
+@plugin.route('/tv/play_by_name/<name>/<season>/<episode>/<lang>', options = {"season": "None", "episode": "None",
+                                                                              "lang": "en"})
 @plugin.route('/tv/play_by_name/<name>/<lang>', name="tv_play_by_name_without_episodes")
```

Comment:
left this one in as having the name and language does happen rather frequently, but the first route wont match this